packages/headless/src/composables/menuButton/createMenuButton.ts

```diff
@@ -32,6 +25,68 @@ export const createMenuButton = createBuilder((options: CreateMenuButtonOptions)
     };
   });
 
+  const getMenu = (el: HTMLElement) => {
+    const id = el.getAttribute("aria-controls") || "";
+    return document.getElementById(id);
+  };
+
+  const focusRelativeItem = (next: "next" | "prev" | "first" | "last") => {
+    const currentMenuItem = document.activeElement as HTMLElement;
+    const currentMenu = currentMenuItem?.closest('[role="menu"]') || getMenu(currentMenuItem);
+    if (!currentMenu) return;
```

Comment:
We don't actually need to get the "aria-controls" value, as we already define the id ourselves:
```suggestion
  const focusRelativeItem = (next: "next" | "prev" | "first" | "last") => {
    const currentMenuItem = document.activeElement as HTMLElement;
    const currentMenu = currentMenuItem?.closest('[role="menu"]') || document.getElementById(menuId);
    if (!currentMenu) return;
```